provisioning/roles/python/tasks/main.yml

```diff
@@ -19,7 +19,7 @@
   with_items:
     - python3-pip
     - python3-all-dev
-    - python3-venv
+    - python3.4-venv
```

Comment:
why is this needed ? Ubuntu ?

The idea behind `python3-venv` is that when a new python version is available, for example 3.5, it gets installed also. I agree it won't be the case in the near future, but why limit ourselves to 3.4 ?